src/Nethermind/Nethermind.Blockchain/BlockTree.cs

```diff
@@ -1624,6 +1623,7 @@ void SetTotalDifficultyDeep(BlockHeader current)
         public event EventHandler<BlockEventArgs>? NewSuggestedBlock;
 
         public event EventHandler<BlockEventArgs>? NewHeadBlock;
+        public event EventHandler<IBlockTree.ForkChoice>? OnForkChoiceUpdated;
```

Comment:
```suggestion
        event EventHandler<OnForkChoiceUpdatedArgs> OnForkChoiceUpdated;
```
to match other events

Reply:
Done